tests/nlu/selectors/test_selectors.py

```diff
@@ -615,6 +615,34 @@ def test_transformer_size_gets_corrected(train_persist_load_with_different_setti
     assert selector.component_config[TRANSFORMER_SIZE] == DEFAULT_TRANSFORMER_SIZE
 
 
+async def test_process_empty_input(
+    create_response_selector: Callable[[Dict[Text, Any]], ResponseSelector],
+    train_and_preprocess: Callable[..., Tuple[TrainingData, List[GraphComponent]]],
+    process_message: Callable[..., Message],
+):
+    pipeline = [
+        {"component": WhitespaceTokenizer},
+        {"component": CountVectorsFeaturizer},
+    ]
+    training_data, loaded_pipeline = train_and_preprocess(
+        pipeline, "data/test_selectors"
+    )
+
+    response_selector = create_response_selector({EPOCHS: 1})
+    response_selector.train(training_data=training_data)
+
+    message = Message(data={TEXT: ""})
+    message = process_message(loaded_pipeline, message)
+
+    classified_message = response_selector.process([message])[0]
+    output = classified_message.get("response_selector").get("default").get("response")
+
+    assert classified_message.get(TEXT) == ""
+    assert not output.get("responses")
+    assert output.get("confidence") == 0.0
+    assert not output.get("intent_response_key")
```

Comment:
Please use existing constants for the strings over here. You'll find them being used in `process` method of `ResponseSelector` class as well.

Reply:
forgot to change this
wIll do now